os/tools/symtab_formatter.c

```diff
@@ -0,0 +1,133 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define ADDR_LENGTH 4
+#define HEXADECIMAL_BASE 16
+#define MAP_FILE_PATH "../build/output/bin/System.map"
+#define BIN_FILE_PATH "../build/output/bin/Symtab.bin"
+#define MAX_NUM_SYM 8000
+#define MAX_SYM_LEN 128
+
+/****************************************************************************
+ * Global Functions
+ ****************************************************************************/
+
+uint32_t symbolnum;
```

Comment:
I think you'd better set this variable to static and initialize to 0 in the start of main

Reply:
Okay. Will modify it .